nes/core/models/entity_type_map.py

```
@@ -3,64 +3,72 @@
-        EntitySubType.SUB_METROPOLITAN_CITY,  # 11 sub-metropolitan cities (उपमहानगरपालिका)
-        EntitySubType.MUNICIPALITY,  # 276 municipalities (नगरपालिका)
-        EntitySubType.RURAL_MUNICIPALITY,  # 460 rural municipalities (गाउँपालिका)
-        EntitySubType.WARD,  # Wards within municipalities (वडा)
-        EntitySubType.CONSTITUENCY,  # Electoral constituencies (निर्वाचन क्षेत्र)
-    },
-    EntityType.PROJECT: {
-        None,  # Project without specific subtype
-        EntitySubType.DEVELOPMENT_PROJECT,  # Development projects (विकास परियोजना)
-    },
-}
+from nes.core.models.entity import EntitySubType, EntityType
```

Flake8 (7.3.0)

[error] 'nes.core.models.entity.EntitySubType' imported but unused

(F401)